### ISSUE TITLE
Is there is reson to use full pouchdb in dependency ?

### ISSUE THREAD

**Issue by zorn-v** [pouchdb-community/ember-pouch#241]
Why not use only `pouchdb-browser` ? In that case there will be no need to compile leveldown which is not needed at all for ember.

**Comment by jlami** [comment]
That does seem alluring, but I'm not sure if it is technically correct. Ember should also be able to work in node, for example with fastboot. I'm not sure if ember-pouch works with fastboot, but using pouchdb-browser would surely prevent this.

We could make the user choose through peer or optional dependencies. But that would make an out of the box installation harder.

**Comment by zorn-v** [comment]
> Ember should also be able to work in node, for example with fastboot.

How ember-data work with fastboot ? Isn't it for client side only ?

**Comment by jlami** [comment]
Yes, it might be better to let people who really want fastboot to have to do more work. But I'm not sure if using pouchdb-browser would make that impossible, or just a little bit harder.

We would have to look into whether pouchdb-browser comes with the required distribution files we use to load pouchdb-find before we can really switch tough.

**Comment by zorn-v** [comment]
https://pouchdb.com/custom.html

**Comment by jlami** [comment]
It does require us to either use pouchdb-browser as a direct dependency or go the peerDependency route. Peer dependencies are technically more correct, but harder to get right for normal users.

Maybe we can use `afterInstall` from ember-cli to setup the default browser config, and let bootstrap users change this default configuration to suite their own needs. It makes it a bit harder to manage the installed versions though.

**Comment by zorn-v** [comment]
> but harder to get right for normal users.

Just one more line in readme/changelog
